shallow_backup.py

```diff
@@ -127,18 +129,19 @@ def copy_dotfolder(dotfolder, backup_path):
 			command = "cp -aRp " + dotfolder + " " + backup_path + "/macOS_Preferences"
 			print(Fore.BLUE + command)
 		elif "Application\ Support" not in dotfolder:
-			command = "cp -aRp " + dotfolder + " " + backup_path + "/" + dotfolder.split("/")[-2]
+			command = "cp -aRp " + dotfolder + " " + \
+					  backup_path + "/" + dotfolder.split("/")[-2]
 		elif "Sublime" in dotfolder:
-			command = "cp -aRp " + dotfolder + " " + backup_path + "/" + dotfolder.split("/")[-3]
+			command = "cp -aRp " + dotfolder + " " + \
+					  backup_path + "/" + dotfolder.split("/")[-3]
 		sp.run(command, shell=True, stdout=sp.PIPE)
 
 
-def copy_dotfile(dotfile, backup_path):
+def _copy_file(source, target):
 	"""
 	Copy dotfile from $HOME.
 	"""
-
-	command = "cp -a " + dotfile + " " + backup_path
+	command = "cp -a '" + source + "' '" + target + "'"
```

Comment:
I think there are nested sets of quotes on this line for some reason.

Reply:
This is on purpose, so the command would look like this:
`cp -a 'source' 'backup_path'` instead of `cp -a source backup_path`.

This avoids to deal with escaping the source/backup_path values. For example if source contains "Application Support" I would have to escape the space character, but when wrapping it in the single quotes I don't need to bother.

Reply:
Good call, I forgot about enclosing quotes!